crates/vhost/src/vhost_kern/vdpa.rs

```diff
@@ -61,6 +61,34 @@ impl<AS: GuestAddressSpace> VhostKernVdpa<AS> {
             backend_features_acked,
         }
     }
+
+    /// Set the addresses for a given vring.
+    ///
+    /// # Arguments
+    /// * `queue_index` - Index of the queue to set addresses for.
+    /// * `config_data` - Vring config data, addresses of desc_table, avail_ring
+    ///     and used_ring are in the guest address space.
+    pub fn set_vring_addr(&self, queue_index: usize, config_data: &VringConfigData) -> Result<()> {
+        if !self.is_valid(config_data) {
+            return Err(Error::InvalidQueue);
+        }
+
+        // vDPA backends expect GPA rather than HVA, so no address
```

Comment:
I would replace GPA with IOVA (that can be mapped 1:1 with GPA, but if there is an iommu can be different).
I would also update the commit description.

Reply:
@stefano-garzarella PR updated. PTAL. Thanks!